src/cryptography/x509/verification.py

```diff
@@ -7,18 +7,24 @@
 import typing
 
 from cryptography.hazmat.bindings._rust import x509 as rust_x509
+from cryptography.x509.base import Certificate
+from cryptography.x509.extensions import Extension, SubjectAlternativeName
 from cryptography.x509.general_name import DNSName, IPAddress
 
 __all__ = [
     "Store",
     "Subject",
+    "VerifiedClient",
+    "ClientVerifier",
     "ServerVerifier",
     "PolicyBuilder",
     "VerificationError",
 ]
 
 Store = rust_x509.Store
 Subject = typing.Union[DNSName, IPAddress]
+VerifiedClient = tuple[Extension[SubjectAlternativeName], list[Certificate]]
```

Comment:
NB: I did this as a type alias for a tuple, but I could make this stronger as a `NewType` if wanted.

Reply:
Please make it a full class -- tuples are huge backwards compat footguns.